tests/integration/targets/mso_schema_site_bd_subnet/tasks/main.yml

```diff
@@ -503,7 +570,7 @@
 - name: Verify non_existing_template
   assert:
     that:
-    - non_existing_template.msg == "Provided template 'non_existing_template' does not exist. Existing templates{{':'}} Template1, Template2, Template4, Template5"
+    - non_existing_template.msg == "Provided condition(s) ['name==non_existing_template'] not matching existing object(s){{':'}} ['name==Template1'], ['name==Template2'], ['name==Template4'], ['name==Template5']"
```

Comment:
This error message is not really user friendly. Why changing it from what it was before?

Reply:
changed because I tried to create a generic message from within the schema class, we can change this message structure